specs/lfp-appeals-api-spec.json

```diff
@@ -0,0 +1,375 @@
+            "description": "Appeal not found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Appeal"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal server error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Appeal"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "Appeal": {
+        "required": [
+          "penaltyIdentifier",
+          "reasons"
+        ],
+        "type": "object",
+        "properties": {
+          "createdBy": {
+            "$ref": "#/components/schemas/CreatedBy"
+          },
+          "penaltyIdentifier": {
+            "$ref": "#/components/schemas/PenaltyIdentifier"
+          },
+          "reasons": {
+            "$ref": "#/components/schemas/Reason"
+          }
+        }
+      },
+      "Attachment": {
+        "required": [
+          "contentType",
+          "id",
+          "name",
+          "size"
+        ],
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "name": {
+            "type": "string"
+          },
+          "contentType": {
+            "type": "string"
+          },
+          "size": {
+            "minimum": 1,
+            "type": "integer",
+            "format": "int32"
+          },
+          "url": {
+            "type": "string"
+          }
+        }
+      },
+      "CreatedBy": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "emailAddress": {
+            "type": "string"
+          }
+        }
+      },
+      "OtherReason": {
+        "required": [
+          "description",
+          "title"
+        ],
+        "type": "object",
+        "properties": {
+          "title": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "attachments": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Attachment"
+            }
+          }
+        }
+      },
+      "IllnessReason": {
+        "required": [
+          "illPerson",
+          "illnessStartDate",
+          "continuedIllness",
+          "illnessEndDate",
+          "illnessImpactFurtherInformation",
+          "attachments"
+        ],
+        "type": "object",
+        "properties": {
+          "illPerson": {
+            "type": "string"
+          },
+          "illnessStartDate": {
+            "type": "string"
+          },
+          "continuedIllness": {
+            "type": "boolean"
+          },
+          "illnessEndDate": {
+            "type": "string"
```

Comment:
Need to specify date format or developers won't know what to submit (for all dates in this spec).

This format has been used in other specs:
```
"illnessEndDate": {
    "type": "string",
    "description": "Date that the illness ended",
    "example": "2021-01-01"
```